### ISSUE TITLE
adorn_totals to accept a vector for name argument when adding row and column totals

### ISSUE THREAD

**Issue by francisbarton** [sfirke/janitor#359]
## Feature request

I wanted to add totals to rows and columns, but to the give them different names, like this:

```
janitor::adorn_totals(where=c("row", "col"), name = c("bracket_total", "area_total"))

Error: Assigned data `name` must be compatible with row subscript `1`.
x 1 row must be assigned.
x Assigned data has 2 rows.
i Only vectors of size 1 are recycled.
```

I think this would be a good feature, when `where` takes both "row" and "col", `name` should also be able to take a vector. Thanks!

**Comment by sfirke** [comment]
Great idea!

**Comment by francisbarton** [comment]
Thank you! I use {janitor} all the time, by the way.
I haven't had a look at the code at all yet - and I am not an experienced programmer - but is this FR something you think you can do relatively easily, or would you welcome me having a crack at an implementation?

**Comment by sfirke** [comment]
Ordinarily I'd say this is a good place to start off. I am trying to move
pretty quickly these days because I'm overdue on submitting this version to
CRAN. On the other hand, I have very little time and can use all the help I
can get.

I appreciate your interest in helping. For starters, do you want to take a
look at the code for adorn_totals? From there I think it might be clear to
you when you play around with the it whether you can make this behavior
work as you intend. If it seems solvable to you, then why don't you fork
this repository, make the changes you can, and submit a pull request and I
will work with you on it. No worries if that feels like too much. And if
you do take the plunge, I can answer questions.

**Comment by francisbarton** [comment]
Yup - I'll have a look and maybe fork and see what I can do. Thank you for the offer of support with it.

**Comment by sfirke** [comment]
@francisbarton do you have interest in working on this?  Could be nice to get it into the next release of janitor

**Comment by francisbarton** [comment]
Hi Sam,
Yes I am interested in this, I just don't know if I have the capability to make the changes. When I looked at it before, I got a bit lost in how it all worked. But that was several months ago. I'll have another look now and get back to you.

**Comment by sfirke** [comment]
closed by #413
